_notices/rsn0047.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 47 # should match notice number
+notice_pin: true # set to true to pin to notice page
+
+title: "Changes to RAPIDS' branching strategy in 25.08"
+notice_author: RAPIDS Ops
+notice_status: In Progress
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Development Process Change
+notice_rapids_version: "v25.08+"
+notice_created: 2025-05-05
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2025-05-05
+---
+
+## Overview
+
+RAPIDS will be migrating to a new branching strategy for the 25.08 release and beyond. The new strategy will use a single long-lived branch called `main` which all pull requests should target. In preparation for release, a release branch will be created from `main` and any remaining pull requests for that release should be merged into the release branch.
+
+Releases and hot fixes will be tagged from the release branch.
+
+This new strategy will be piloted on several RAPIDS libraries (`rapidsai/rmm`, `rapidsai/kvikio`, and `rapidsia/cugraph-gnn`) during the `25.08` development cycle. They will migrate starting shortly after the `25.06` release. After the `25.08` release, the new strategy will be implemented across all RAPIDS libraries.

```

Comment:
Small typo rapidsia -> rapidsai on cugraph dir. Other small nits feel free to ignore.  I also added break lines to make easier the review per line instead of paragraph. 

```suggestion
This new strategy will be piloted on several RAPIDS libraries (`rapidsai/rmm`, 
`rapidsai/kvikio`, and `rapidsai/cugraph-gnn`) during the `25.08` development cycle.
Migrations will begin shortly after the 25.06 release. After the `25.08` release,
the new strategy will be implemented across all RAPIDS libraries.
```